tests/integration/test_projectconfigs.py

```diff
@@ -389,29 +389,21 @@ def test_cached_project_config(mini_sentry, relay):
         }
     )
 
-    # Caches must be expired at this point, and we are in the grace period.
-    time.sleep(2)
-    # The state must be stale and still be valid, but the update will be scheduled to get the new project state.
-    data = get_response(relay, packed, signature)
-    assert data["configs"][public_key]["projectId"] == project_key
-    assert not data["configs"][public_key]["disabled"]
-
-    # This is still a grace period, and the state for us must be still valid, even though we get the error parsing the new state.
     try:
         # Give it a bit time for update to go through.
-        time.sleep(1)
+        time.sleep(3)
         data = get_response(relay, packed, signature)
+        assert data["configs"][public_key]["disabled"]
+
+        from pprint import pprint
```

Comment:
Why this import? What happened to the asserts in this test?

Reply:
Removed, used for some debugging. And also reverted test